release_bot/configuration.py

```diff
@@ -42,6 +42,7 @@ def __init__(self):
         self.github_app_installation_id = ''
         self.github_app_id = ''
         self.github_app_cert_path = ''
+        self.clone_url = ''
```

Comment:
Please document this in readme.

Reply:
I've updated the README. Any changes required in the documentation?